src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -63,24 +43,8 @@ public class GraphQLHttpMiddleware<TSchema> : GraphQLHttpMiddleware
         IDocumentExecuter<TSchema> documentExecuter,
         IServiceScopeFactory serviceScopeFactory,
         GraphQLHttpMiddlewareOptions options,
-        IServiceProvider provider,
         IHostApplicationLifetime hostApplicationLifetime)
-        : this(next, serializer, documentExecuter, serviceScopeFactory, options,
-              CreateWebSocketHandlers(serializer, documentExecuter, serviceScopeFactory, provider, hostApplicationLifetime, options))
-    {
-    }
-
-    /// <summary>
-    /// Initializes a new instance.
-    /// </summary>
-    protected GraphQLHttpMiddleware(
-        RequestDelegate next,
-        IGraphQLTextSerializer serializer,
-        IDocumentExecuter<TSchema> documentExecuter,
-        IServiceScopeFactory serviceScopeFactory,
-        GraphQLHttpMiddlewareOptions options,
-        IEnumerable<IWebSocketHandler<TSchema>>? webSocketHandlers = null)
-        : base(next, serializer, options, webSocketHandlers)
+        : base(next, serializer, options, hostApplicationLifetime)
```

Comment:
Technically `hostApplicationLifetime` _may_ be optional. At least this is nothing more than an additional source of information about cancellation.

Reply:
Although I forget (again) that the project is called Transports.**AspNetCore** so there is always should be `hostApplicationLifetime`.